core-sdk-samples/higgs-shop-sample-app/src/hooks/useLocalStorage.ts

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable no-console */
+import { Dispatch, SetStateAction, useState } from 'react';
+
+type SetValue<T> = Dispatch<SetStateAction<T>>;
```

Comment:
This is just to mirror the API for `setState` so that `useLocalStorage` behaves similar to `setState`. The gist is that second attribute provided by `localStorage` is a function that conforms to the `SetStateAction` interface that can in turn be put into a `Dispatch` function.